docs/security/trust/content_trust.md

```diff
@@ -221,16 +221,16 @@ Once you have a root key on your system, subsequent images repositories
 you create can use that same root key:
 
 ```bash
-$ docker push docker.io/docker/seaside:latest
-The push refers to a repository [docker.io/docker/seaside] (len: 1)
+$ docker push docker.io/<username>/<someimage>:latest
```

Comment:
Here we are pushing it to a different image repo but wouldn't it still be signed with the root key as the lines after this example mention? Maybe we can just change the name to something else other than nottrusttest/trusttest. Any suggestion?